### PR TITLE
Language change: workflows orchestrate, not just schedule

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Change descriptions of workflows to "orchestrate" where relevant

# Reasons
https://circleci.atlassian.net/browse/DOCTEAM-800?atlOrigin=eyJpIjoiZWQ1MjNlMzQxOTYzNGZkYmE2NTU2NzlhZjJlMWY0ZTkiLCJwIjoiaiJ9

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/config-intro.adoc [hunk 1]
```diff
@@ -372,6 +372,6 @@ Using what you have learned above, you are ready to create some powerful pipelin
 
 * <<configuration-reference#,Configuring CircleCI>>
 * <<executor-intro#,Executors and Images>>
-* <<workflows#,Using Workflows to Schedule Jobs>>
+* <<workflows#,Using Workflows to Orchestrate Jobs>>
```
Comment (devinmarieb) [inline jekyll/_cci2/config-intro.adoc:375]:
```suggestion
* <<workflows#,Using workflows to orchestrate jobs>>
```

File: jekyll/_cci2/config-intro.adoc [hunk 2]
```diff
@@ -372,6 +372,6 @@ Using what you have learned above, you are ready to create some powerful pipelin
 
 * <<configuration-reference#,Configuring CircleCI>>
 * <<executor-intro#,Executors and Images>>
```
Comment (devinmarieb) [inline jekyll/_cci2/config-intro.adoc:374]:
```suggestion
* <<executor-intro#,Executors and images>>
```

File: jekyll/_cci2/configuration-reference.md [hunk 3]
```diff
@@ -2021,7 +2021,7 @@ workflows:
 ## **`workflows`**
 {: #workflows }
 
-Used for orchestrating all jobs. Each workflow consists of the workflow name as a key and a map as a value. A name should be unique within the current `config.yml`. The top-level keys for the Workflows configuration are `version` and `jobs`. For more information, see the [Using Workflows to Schedule Jobs]({{site.baseurl}}/workflows/) page.
+Used for orchestrating all jobs. Each workflow consists of the workflow name as a key and a map as a value. A name should be unique within the current `config.yml`. The top-level keys for the Workflows configuration are `version` and `jobs`. For more information, see the [Using Workflows to Orchestrate Jobs]({{site.baseurl}}/workflows/) page.
```
Comment (devinmarieb) [inline jekyll/_cci2/configuration-reference.md:2024]:
```suggestion
Used for orchestrating all jobs. Each workflow consists of the workflow name as a key and a map as a value. A name should be unique within the current `.circleci/config.yml`. The top-level keys for the workflows configuration are `version` and `jobs`. For more information, see the [Using workflows to orchestrate jobs]({{site.baseurl}}/workflows/) page.
```

File: jekyll/_cci2/configuration-reference.md [hunk 4]
```diff
@@ -2230,7 +2230,7 @@ context | N | String/List | The name of the context(s). The initial default name
 ###### **`type`**
 {: #type }
 
-A job may have a `type` of `approval` indicating it must be manually approved before downstream jobs may proceed. For more information see the [Using Workflows to Schedule Jobs]({{site.baseurl}}/workflows/#holding-a-workflow-for-a-manual-approval) page.
+A job may have a `type` of `approval` indicating it must be manually approved before downstream jobs may proceed. For more information see the [Using Workflows to Orchestrate Jobs]({{site.baseurl}}/workflows/#holding-a-workflow-for-a-manual-approval) page.
```
Comment (devinmarieb) [inline jekyll/_cci2/configuration-reference.md:2233]:
```suggestion
A job may have a `type` of `approval`, indicating it must be manually approved before downstream jobs may proceed. For more information see the [Using workflows to orchestrate jobs]({{site.baseurl}}/workflows/#holding-a-workflow-for-a-manual-approval) page.
```

File: jekyll/_cci2/ecs-ecr.md [hunk 5]
```diff
@@ -125,7 +125,7 @@ workflows:
           container-image-name-updates: "container=${AWS_RESOURCE_NAME_PREFIX}-service,tag=${CIRCLE_SHA1}"
 ```
 
-Note the use of Workflows to define job run order/concurrency. See the [Using workflows to schedule jobs]({{site.baseurl}}/workflows/) page for more information.
+Note the use of Workflows to define job run order/concurrency. See the [Using Workflows to Orchestrate Jobs]({{site.baseurl}}/workflows/) page for more information.
```
Comment (devinmarieb) [inline jekyll/_cci2/ecs-ecr.md:128]:
```suggestion
Note the use of workflows to define job run order/concurrency. See the [Using workflows to orchestrate jobs]({{site.baseurl}}/workflows/) page for more information.
```

File: jekyll/_cci2/gitlab-integration.adoc [hunk 6]
```diff
@@ -457,7 +457,7 @@ GitLab-based projects do not have the following system environment variables ava
 | The SHA1 hash of the last commit of the current build.
 
 | `CIRCLE_TAG`
-| The name of the git tag, if the current build is tagged. For more information, see the <<workflows#executing-workflows-for-a-git-tag,Git Tag Job Execution section>> of the Using Workflows to Schedule Jobs document. 
+| The name of the git tag, if the current build is tagged. For more information, see the <<workflows#executing-workflows-for-a-git-tag,Git Tag Job Execution section>> of the Using Workflows to Orchestrate Jobs document.
```
Comment (devinmarieb) [inline jekyll/_cci2/gitlab-integration.adoc:460]:
```suggestion
| The name of the git tag, if the current build is tagged. For more information, see the <<workflows#executing-workflows-for-a-git-tag,Git tag job execution>> section of the Using workflows to orchestrate jobs document.
```

File: jekyll/_cci2/language-javascript.md [hunk 7]
```diff
@@ -155,7 +155,7 @@ workflows:
 ### 4. Add jobs to the workflow
 {: #add-jobs-to-the-workflow }
 
-Now that we have our workflow, `build_test_deploy`, we can use it to orchestrate the running of our `build_and_test` and `deploy` jobs. Refer to the [Using Workflows to Schedule Jobs]({{site.baseurl}}/workflows/) page for more details about orchestrating jobs with concurrent, sequential, and manual approval workflows.
+Now that we have our workflow, `build_test_deploy`, we can use it to orchestrate the running of our `build_and_test` and `deploy` jobs. Refer to the [Using Workflows to Orchestrate Jobs]({{site.baseurl}}/workflows/) page for more details about orchestrating jobs with concurrent, sequential, and manual approval workflows.
```
Comment (devinmarieb) [inline jekyll/_cci2/language-javascript.md:158]:
```suggestion
Now that we have our workflow, `build_test_deploy`, we can use it to orchestrate the running of our `build_and_test` and `deploy` jobs. Refer to the [Using workflows to orchestrate jobs]({{site.baseurl}}/workflows/) page for more details about orchestrating jobs with concurrent, sequential, and manual approval workflows.
```

File: jekyll/_cci2/language-python.md [hunk 8]
```diff
@@ -127,7 +127,7 @@ workflows:
 ### 5. Add jobs to the workflow
 {: #add-jobs-to-the-workflow }
 
-Now that we have our workflow, `build_test_deploy`, we can use it to orchestrate the running of our `build_and_test` and `deploy` jobs. Refer to the [Using Workflows to Schedule Jobs]({{site.baseurl}}/workflows/) page for more details about orchestrating jobs with concurrent, sequential, and manual approval workflows.
+Now that we have our workflow, `build_test_deploy`, we can use it to orchestrate the running of our `build_and_test` and `deploy` jobs. Refer to the [Using Workflows to Orchestrate Jobs]({{site.baseurl}}/workflows/) page for more details about orchestrating jobs with concurrent, sequential, and manual approval workflows.
```
Comment (devinmarieb) [inline jekyll/_cci2/language-python.md:130]:
```suggestion
Now that we have our workflow, `build_test_deploy`, we can use it to orchestrate the running of our `build_and_test` and `deploy` jobs. Refer to the [Using workflows to orchestrate jobs]({{site.baseurl}}/workflows/) page for more details about orchestrating jobs with concurrent, sequential, and manual approval workflows.
```

File: jekyll/_cci2/notify-a-slack-channel-of-a-paused-workflow.adoc [hunk 9]
```diff
@@ -19,7 +19,7 @@ Slack is a real-time collaboration application where team members can work toget
 
 The https://circleci.com/developer/orbs/orb/circleci/slack[CircleCI Slack orb] enables you to create different notifications and messages that can be delivered to your desired recipients. 
 
-This page shows how to use a built-in job from the Slack orb to notify a Slack channel when there is a paused workflow awaiting approval. For more information on approval see the <<workflows#holding-a-workflow-for-a-manual-approval,Using Workflows to Schedule Jobs>> page.
+This page shows how to use a built-in job from the Slack orb to notify a Slack channel when there is a paused workflow awaiting approval. For more information on approval see the <<workflows#holding-a-workflow-for-a-manual-approval,Using Workflows to Orchestrate Jobs>> page.
```
Comment (devinmarieb) [inline jekyll/_cci2/notify-a-slack-channel-of-a-paused-workflow.adoc:22]:
```suggestion
This page shows how to use a built-in job from the Slack orb to notify a Slack channel when there is a paused workflow awaiting approval. For more information on approval see the <<workflows#holding-a-workflow-for-a-manual-approval,Using workflows to orchestrate jobs>> page.
```

File: jekyll/_cci2/notify-a-slack-channel-of-a-paused-workflow.adoc [hunk 10]
```diff
@@ -75,7 +75,7 @@ workflows:
 
 === 5. Add jobs to your workflow
 
-Now that we have our workflow, `test-hold-deploy`, we can use it to orchestrate the running of our `test` and `deploy` jobs with a pause requiring manual approval in-between. Refer to the <<workflows#,Using Workflows to Schedule Jobs>> page for more details about orchestrating jobs with concurrent, sequential, and manual approval workflows.
+Now that we have our workflow, `test-hold-deploy`, we can use it to orchestrate the running of our `test` and `deploy` jobs with a pause requiring manual approval in-between. Refer to the <<workflows#,Using Workflows to Orchestrate Jobs>> page for more details about orchestrating jobs with concurrent, sequential, and manual approval workflows.
```
Comment (devinmarieb) [inline jekyll/_cci2/notify-a-slack-channel-of-a-paused-workflow.adoc:78]:
```suggestion
Now that we have our workflow, `test-hold-deploy`, we can use it to orchestrate the running of our `test` and `deploy` jobs with a pause requiring manual approval in-between. Refer to the <<workflows#,Using workflows to orchestrate jobs>> page for more details about orchestrating jobs with concurrent, sequential, and manual approval workflows.
```

File: jekyll/_cci2/triggers-overview.adoc [hunk 11]
```diff
@@ -71,6 +71,6 @@ Scheduled pipelines allow you to trigger pipelines periodically based on a sched
 [#next-steps]
 == Next steps
 
-Read about orchestrating elements of work within pipelines on the <<workflows#,Using Workflows to Schedule Jobs>> page.
+Read about orchestrating elements of work within pipelines on the <<workflows#,Using Workflows to Orchestrate Jobs>> page.
```
Comment (devinmarieb) [inline jekyll/_cci2/triggers-overview.adoc:74]:
```suggestion
Read about orchestrating elements of work within pipelines on the <<workflows#,Using workflows to orchestrate jobs>> page.
```

File: jekyll/_cci2/workflows.md [hunk 12]
```diff
@@ -1,9 +1,7 @@
 ---
 layout: classic-docs
-title: "Using Workflows to Schedule Jobs"
-short-title: "Using Workflows to Schedule Jobs"
-description: "Using Workflows to Schedule Jobs"
-order: 30
+title: "Using Workflows to Orchestrate Jobs"
```
Comment (devinmarieb) [inline jekyll/_cci2/workflows.md:3]:
```suggestion
title: "Using workflows to orchestrate jobs"
```

File: jekyll/_cci2/workspaces.md [hunk 13]
```diff
@@ -230,7 +230,7 @@ It is important to define paths and files when using `persist_to_workspace`. Not
 {: #see-also }
 {:.no_toc}
 
-- For conceptual and usage information on Workflows, see the [Using Workflows to Schedule Jobs]({{site.baseurl}}/workflows) page.
+- For conceptual and usage information on Workflows, see the [Using Workflows to Orchestrate Jobs]({{site.baseurl}}/workflows) page.
```
Comment (devinmarieb) [inline jekyll/_cci2/workspaces.md:233]:
```suggestion
- For conceptual and usage information on Workflows, see the [Using workflows to orchestrate jobs]({{site.baseurl}}/workflows) page.
```

**Review by devinmarieb — APPROVED** [review]
The content is great, I just made formatting suggestions since I thought we were moving away from title case.

**Comment by rosieyohannan** [comment]
> The content is great, I just made formatting suggestions since I thought we were moving away from title case.

Thanks Devin! In our style guide we still say to use title case for referencing headings in prose: https://circleci.com/docs/style/headings/ I am happy to change it though. I'll merge this as is, and then we can decide.

**Pull request merged**